README.md

```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My-Portfolio
 
-> Description the project.
+> Setup and mobile version skeleton
 
+![screenshot](./Images/Setup-mobile-skeleton.png)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML/CSS
+- LINTERS
+- VSCODE
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+- GITHUB
+- IDE
 
+### Setup
 
+- git clone https://github.com/thenarain/Portfolio.git
```

Comment:
[OPTIONAL]

- [ ] You can give users more detailed steps on how to clone and use your repo.

Reply:
Thanks for the suggestion. I have done the required changes.